environments/operator/obs-operator.jsonnet

```diff
@@ -1,3 +1,18 @@
-(import '../../components/observatorium.libsonnet') + {
-  config+:: (import 'operator-config.libsonnet'),
-} + (import '../../components/observatorium-configure.libsonnet')
+local config = import 'operator-config.libsonnet';
+local manifests = ((import '../../components/observatorium.libsonnet') + {
+                     config+:: config,
+                   } + (import '../../components/observatorium-configure.libsonnet')).manifests;
+
+local addOwner(k, v) = v { metadata+: { ownerReferences: [{
+  apiVersion: config.apiVersion,
+  blockOwnerdeletion: true,
+  controller: true,
+  kind: config.kind,
+  name: config.name,
+  uid: config.uid,
+}] } };
+local ownermanifests = std.mapWithKey(addOwner, manifests);
+
+{
+  manifests: ownermanifests,
```

Comment:
I would inline the `std.map` call here, no need for the local variables above except the imports

Reply:
Done